test.md

```diff
@@ -0,0 +1 @@
+#This file should be ignored by git

```

Comment:
- [ ] Kindly make sure that this file is actually ignored by git. It shouldn't be in the GitHub repo! 🤓 🛠️